### PR TITLE
ZCS-10872 Copyright displayed at top, Setting in Pref -> Mail not translated

### PR DESCRIPTION
**Pull request by miteshsavani810**
- Added translation of `Preferences -> Mail -> "Always display received time in email list"`
- `Issue`: Copyright text display at Top. `Expected`: Copyright text should be visible at the bottom of screen

### CODE REVIEW

File: WebRoot/messages/ZmMsg_ja.properties [hunk 1]
```diff
@@ -979,6 +979,7 @@ displayIM = IM\u30c1\u30e3\u30c3\u30c8\u306e\u8868\u793a\u65b9\u6cd5\u3092\u6307
 displayExternalImages = \u30a4\u30e1\u30fc\u30b8\u3092\u8868\u793a
 displayMail = \u53d7\u4fe1\u30e1\u30fc\u30eb\u306e\u8868\u793a\u5f62\u5f0f\uff1a
 displayMailToolTip = \u30e1\u30fc\u30eb\u306e\u8868\u793a\u65b9\u6cd5\u3092\u6307\u5b9a
+displayTimeInMailList = \u30e1\u30fc\u30ea\u30f3\u30b0 \u30ea\u30b9\u30c8\u306b\u53d7\u4fe1\u6642\u523b\u3092\u5e38\u306b\u8868\u793a\u3059\u308b
```
Comment (k-kato) [inline WebRoot/messages/ZmMsg_ja.properties:982]:
```suggestion
displayTimeInMailList = \u30e1\u30fc\u30eb\u4e00\u89a7\u3067\u53d7\u4fe1\u6642\u523b\u3092\u5e38\u306b\u8868\u793a\u3059\u308b
```
```
メーリング リストに受信時刻を常に表示する -> メール一覧で受信時刻を常に表示する
```

**Review by k-kato — COMMENTED** [review]
Please fix Japanese translation.

**Review by silentsakky — APPROVED** [review]

File: WebRoot/messages/ZmMsg_ja.properties (hunk 1 above)
Comment (k-kato) [inline WebRoot/messages/ZmMsg_ja.properties:982]:
```suggestion
displayTimeInMailList = \u30e1\u30fc\u30eb\u4e00\u89a7\u3067\u53d7\u4fe1\u65e5\u6642\u3092\u5e38\u306b\u8868\u793a\u3059\u308b
```
```
メーリング リストに受信時刻を常に表示する -> メール一覧で受信日時を常に表示する
```

Comment (miteshsavani810) [reply WebRoot/messages/ZmMsg_ja.properties]: Fixed.

**Review by k-kato — APPROVED** [review]

File: WebRoot/messages/ZmMsg_ja.properties (hunk 1 above)
Comment (k-kato) [reply WebRoot/messages/ZmMsg_ja.properties]: Thanks!

**Comment by ghen2** [comment]
> * Added translation of `Preferences -> Mail -> "Always display received time in email list"`

`timeInMailListChangeRestartShow` and `timeInMailListChangeRestartHide` are related missing translations.
(this text is displayed in a popup after changing that setting & saving, suggesting to reload the UI)

**Comment by silentsakky** [comment]
> > * Added translation of `Preferences -> Mail -> "Always display received time in email list"`
> 
> `timeInMailListChangeRestartShow` and `timeInMailListChangeRestartHide` are related missing translations. (this text is displayed in a popup after changing that setting & saving, suggesting to reload the UI)

This is fixed in https://github.com/Zimbra/zm-web-client/pull/813 and will be released in the next patch.
For quicker turnaround time it's requested to ask these questions at https://forums.zimbra.org/ so everyone can read it and work on it.

**Pull request merged**